### PR TITLE
Added Reasons

### PR DESCRIPTION
**Pull request by Scuph**
^^

### CODE REVIEW

**Comment by Wild1145** [comment]
That doesnt look right...

**Comment by TheRevalation** [comment]
@Wild1145 Some of it could be wrong, but overall should work

**Comment by Wild1145** [comment]
@TheRevalation  It might just be down to me being really tired right now, but it looks really silly where you have two different messages in two different methods. I could just be missing something obvious, but it looks out of the ordinary.

**Comment by JeromSar** [comment]
@TheRevalation Please find a better way to combine the two nearly identical methods. :)

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java [hunk 1]
```diff
@@ -10,13 +10,13 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername]")
+@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername] [reason]")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java]: This should be "/<command> <player> [reason]"

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java [hunk 2]
```diff
@@ -28,15 +28,46 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
             playerMsg(TFM_Command.PLAYER_NOT_FOUND);
             return true;
         }
+        else if (args.length > 1)
+        {
+            final String reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+            smite(player, reason);
+            return true;
+        }
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java]: You're missing curly braces. Did you *test* before making a PR?

**Comment by TheRevalation** [comment]
@jeromsar accidentally deleted that, whoops.

**Comment by TheRevalation** [comment]
@jeromsar close this for now and I'll fix this up.

**Comment by JeromSar** [comment]
@TheRevalation No need to close this. Just append your commits to this PR.

**Comment by JeromSar** [comment]
Closing for inactivity.

**Comment by Scuph** [comment]
I have a new PR open for this >.<

and this is a PR @JeromSar! You silly goose.

**Comment by JeromSar** [comment]
@Scuph Fixed :P

**Pull request closed without merging**